cmd/config/get.go

```diff
@@ -0,0 +1,70 @@
+package config
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pingidentity/pingctl/internal/logger"
+	"github.com/pingidentity/pingctl/internal/output"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"gopkg.in/yaml.v3"
+)
+
+func init() {
+	l := logger.Get()
+
+	l.Debug().Msgf("Initializing Config Get Subcommand...")
+}
+
+func NewConfigGetCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "get",
+		Short: "Get pingctl configuration settings.",
+		Long:  `Get pingctl configuration settings.`,
```

Comment:
Might be good to have an example of the command here and in the other subcommand descriptions